debtransform

```diff
@@ -233,17 +233,25 @@ sub dotar_quilt {
   my $input_file_name = basename($input_file_path_without_ext);
   my $decompress_dst = "${tempdirname}/${input_file_name}.tar";
 
-  # print "Decompress '$decompress_src' to '$decompress_dst' for concatenation\n";
-
-  if (($ext eq ".tgz") or ($ext eq ".tar.gz")) {
-      system("gzip -dck \"$decompress_src\" > \"$decompress_dst\"") == 0 || die("Can not unpack gzip archive '$decompress_src': $!\n");
-  } elsif (($ext eq ".tar.xz")) {
-      system("xz -dck \"$decompress_src\" > \"$decompress_dst\"") == 0 || die("Can not unpack xz archive '$decompress_src': $!\n");
-  } elsif (($ext eq ".tar.bz2")) {
-      system("bzip2 -dck \"$decompress_src\" > \"$decompress_dst\"") == 0 || die("Can not unpack bzip2 archive '$decompress_src': $!\n");
-  }
+  my $tar_for_concatination_path;
+
+  if (($ext eq ".tar")) {
+      $tar_for_concatination_path = $input_file_path;
```

Comment:
I wonder if it wouldn't be easier to just do a 
```
  system('cp', $decompress_src, $decompress_dst);
```
here to keep the code simple.

Reply:
Good idea. It simplifies code reading.